source/panic_fake/fake_panic.go

```diff
@@ -19,8 +19,10 @@ package panic_fake
 // Source implements FeatureSource.
 type Source struct{}
 
+// Name returns a friendly name for this source.
```

Comment:
Maybe change this description to `Name returns a identifier string for this feature source`?